src/SDKs/Authorization/Authorization.Tests/Authorization.Tests.csproj

```diff
@@ -10,9 +10,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Authorization" Version="2.5.0-preview" />
+    <!--<PackageReference Include="Microsoft.Azure.Management.Authorization" Version="2.5.0-preview" />-->
```

Comment:
@darshanhs90 remove commented code.
Also start using latest version of ResourceManager to record all your tests. The latest version is 1.6.0 for ResourceManager nuget package.

Reply:
Resolved it

---
In reply to: [153349537](https://github.com/Azure/azure-sdk-for-net/pull/3872#discussion_r153349537) [](ancestors = 153349537)